eosclubhouse/system.py

```diff
@@ -43,6 +44,18 @@ class Desktop:
     SETTINGS_HACK_MODE_KEY = 'HackModeEnabled'
     SETTINGS_HACK_ICON_PULSE = 'HackIconPulse'
 
+    # Apps ids to override flatpak GTK3_MODULES with libclippy
+    CLIPPY_APPS = [
+        'com.endlessm.dinosaurs.en',
```

Comment:
All the other apps have as well a corresponding com.endlessm file. We should remove that too.

Reply:
Okay, I've added all the ids.